### ISSUE TITLE
Unable to navigate to datepicker calendar button with screen reader

### ISSUE THREAD

**Issue by Blackbaud-LouisPujol** [blackbaud/skyux-datetime#55]
### Expected behavior
Using a screen reader, I can move from the input box of the datepicker to the calendar button.

### Actual behavior
Using a screen reader, I move from the input box of the datepicker to the next element on the page and skip the calendar button.

### Steps to reproduce
*Note: I am using VoiceOver on a Mac with Safari*
Go to the [datepicker demo](https://developer.blackbaud.com/skyux/components/datepicker)
Navigate to the datepicker input box using the screen reader
Navigate to the next element which should be the calendar button

### Impact
For clients using a screen reader this means that the user must manually input a date in the datepicker and cannot use the calendar option. This experience is different from the timepicker which does allow you to highlight the timepicker button.

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-MattGregg Is the current behavior intentional, or do you think this is a bug?

**Comment by Blackbaud-MattGregg** [comment]
@Blackbaud-SteveBrush Thinking this might be a bug. I think I recall this used to work this way. It does in SKY UX 1. I don't think it is a high priority issue though. Screen reader and keyboard users can still input dates using their keyboards.

**Comment by Blackbaud-AlexKingman** [comment]
@Blackbaud-LouisPujol it looks like this issue has been resolved with a prior PR, as you can now navigate to the calendar button using the tab key. Enter/Space keys will activate the calendar and then the user can tab into the calendar and select dates using purely keyboard controls.

Closing for now. Please reopen if you still feel there is more work to do here. Thanks!
